### PR TITLE
Add Deutsche Bank

### PR DESCRIPTION
**Pull request by phallobst**

### CODE REVIEW

**Review by Jawshy — COMMENTED** [review]
After [translating](https://www.deepl.com/translator), I can see that SMS 2FA is by using mobileTAN and proprietary 2FA is by using the photoTAN app. However, I couldn't see any hardware 2FA.

**Comment by phallobst** [comment]
Using the photoTAN method is not only possible via the app, but also with a dedicated hardware device. My guess is that the bank is trying to guide the customer to use the app, so it is rather hidden. There is a manual (in German) available:
https://www.deutsche-bank.de/content/dam/deutschebank/de/shared/pdf/pk-digital-banking-photo-tan-bedienungsanleitung-lesegeraet.pdf

**Comment by Jawshy** [comment]
Thanks for the link. Do you know if the photoTAN reader is readily available to customers?

**Comment by phallobst** [comment]
Yes, you can order them via the bank at a low price (or even for free if you have accounts of a certain type).

**Review by Jawshy — APPROVED** [review]

File: _data/banking.yml [hunk 1]
```diff
@@ -394,6 +394,15 @@ websites:
...
 
+  - name: Deutsche Bank
+    url: https://www.deutsche-bank.de/
+    img: deutschebank.png
+    tfa:
+      - sms
+      - proprietary
+      - hardware
+    doc: https://www.deutsche-bank.de/pk/lp/psd2.html
+
```
Comment (Carlgo11) [inline _data/banking.yml:453]:
```suggestion
    regions:
      - de

```

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**